CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ nav_order: 1
 
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
 
+- Fix `aiven_kafka_topic`: add validation for the `partitions` field
```

Comment:
Please leave a note what a "fix" you've made.
And I think this is not a bug, because it works properly. You have added an improvement (preventive validation).